### ISSUE TITLE
Potential Issue with EncoderDecoder Model with CrossEntropy Loss

### ISSUE THREAD

**Issue by gkeskin07** [openspeech-team/openspeech#172]
Hi,

I see that the LSTM Attention Decoder takes the log_softmax of the step outputs [inside the model](https://github.com/openspeech-team/openspeech/blob/main/openspeech/decoders/lstm_attention_decoder.py#L155). However, cross entropy loss uses [nn.CrossEntropyLoss](https://github.com/openspeech-team/openspeech/blob/main/openspeech/criterion/cross_entropy/cross_entropy.py#L108), which takes another log_softmax inside. Shouldn't `nn.NLLLoss` be used instead of `nn.CrossEntropyLoss`? This would cause issues in models such as `ConformerLSTMModel`.

**Comment by sooftware** [comment]
As far as I know, nn.CrossEntropy measure whether log_softmax is applied already and decide whether to apply or not.

**Comment by upskyy** [comment]
Also, because `log_softmax(log_softmax(x)) = log_softmax(x)`, the result is the same.
